src/Icicle/Sea/FromAvalanche/Psv.hs

```diff
@@ -148,21 +143,22 @@ seaOfAllocProgram mode state
 
 ------------------------------------------------------------------------
 
-seaOfCollectFleet :: PsvMode -> [SeaProgramState] -> Doc
-seaOfCollectFleet mode states
+seaOfCollectFleet :: [SeaProgramState] -> Doc
+seaOfCollectFleet states
  = vsep
  [ "#line 1 \"collect fleet state\""
  , "static void psv_collect_fleet (ifleet_t *fleet)"
  , "{"
- , "    imempool_t *into_pool   = imempool_create ();"
- , "    imempool_t *last_pool   = fleet->mempool;"
- , "    iint_t      chord_count = fleet->chord_count;"
+ , "    imempool_t *into_pool       = imempool_create ();"
+ , "    imempool_t *last_pool       = fleet->mempool;"
+ , "    iint_t      max_chord_count = fleet->max_chord_count;"
+ , "    iint_t      chord_count     = fleet->chord_count;"
```

Comment:
ah. so this is the number of chords on the current entity - so we only need to garbage collect the first chord_count states?